libraries/AP_Math/AP_Math.h

```diff
@@ -277,6 +277,14 @@ Vector3f rand_vec3f(void);
 // return true if two rotations are equal
 bool rotation_equal(enum Rotation r1, enum Rotation r2) WARN_IF_UNUSED;
 
+/*
+ * return a velocity correction (in m/s in NED) for a sensor's position given it's position offsets
+ * sensor_offset_bf is in meters in body frame (Foward, Right, Down)
+ * rot_ef_to_bf is a rotation matrix to rotate from earth-frame (NED) to body frame
+ * angular_rate is rad/sec
```

Comment:
Add another line to the comment to indicate that the correction should added to the sensor NED measurement (not subtracted).

Reply:
Thanks, resolved in a follow-up commit.